cfgmgr/buffermgrdyn.cpp

```diff
@@ -1706,16 +1783,17 @@ task_process_status BufferMgrDynamic::handleOneBufferPgEntry(const string &key,
         if (!ignored && bufferPg.lossless)
         {
             doUpdatePgTask(key, port);
-
-            if (!bufferPg.configured_profile_name.empty())
-            {
-                m_bufferProfileLookup[bufferPg.configured_profile_name].port_pgs.insert(key);
-            }
         }
         else
         {
             SWSS_LOG_NOTICE("Inserting BUFFER_PG table entry %s into APPL_DB directly", key.c_str());
             m_applBufferPgTable.set(key, fvVector);
+            bufferPg.running_profile_name = bufferPg.configured_profile_name;
+        }
+
+        if (!bufferPg.configured_profile_name.empty())
```

Comment:
Add some comments: the motivation is to record the lossy PG so that it can be removed when the port is admin down